### DISCUSSION TITLE
Comment définir la sonnerie?

### DISCUSSION THREAD

**Discussion by EdwardEric22** [jagrosh/MusicBot#1377] [General]
Si vous avez un compte Apple Music, vous pouvez utiliser le service pour télécharger des sonneries à partir de n'importe quelle chanson. iTunes propose différents formats audio et vous pouvez choisir celui qui correspond à vos préférences. Faites simplement glisser la sonnerie souhaitée vers l'icône Sound Studio dans le Finder. Sélectionnez la piste audio et cliquez sur le bouton de téléchargement. Ensuite, votre nouvelle sonnerie est prête à être utilisée. C'est très facile! Vous pouvez également utiliser un logiciel d'édition de musique gratuit, tel qu'Audacity, pour créer des [sonnerie telephone](https://telechargersonneries.com/).

**Comment by Massimiliano111** [comment]
[Scarica suonerie gratis](https://e-suonerie.com/) e aggiungi un tocco personale a ogni chiamata. Troverai una vasta selezione di suonerie gratuite per ogni gusto musicale. Che tu preferisca melodie rilassanti, ritmi coinvolgenti o suoni divertenti, troverai sicuramente qualcosa che ti piace. Non importa se sei un amante del pop, del rock o della musica classica, con le suonerie gratuite puoi esprimere la tua personalità in modo unico.
